### PR TITLE
corrected docs of README.md

### PR DESCRIPTION
**Pull request by iamrajiv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ docker run -e TERM -it --rm archlinux/base bash -uexc '
   exec zsh'
 ```
 
-## Is it really fast?
+## Is it fast?
```
Comment (romkatv) [inline README.md]: Why this change?

File: README.md [hunk 2]
```diff
@@ -231,17 +231,17 @@ if [[ -r "${XDG_CACHE_HOME:-$HOME/.cache}/p10k-instant-prompt-${(%):-%n}.zsh" ]]
 fi
 ```
 
-It's important that you copy the lines verbatim. Don't replace `source` with something else, don't
+You must copy the lines verbatim. Don't replace `source` with something else, don't
```
Comment (romkatv) [inline README.md]: Why?

File: README.md [hunk 3]
```diff
@@ -231,17 +231,17 @@ if [[ -r "${XDG_CACHE_HOME:-$HOME/.cache}/p10k-instant-prompt-${(%):-%n}.zsh" ]]
...
 call `zcompile`, don't redirect output, etc.
 
-When instant prompt is enabled, for the duration of zsh initialization standard input is redirected
-to `/dev/null` and standard output with standard error are redirected to a temporary file. Once zsh
+When the instant prompt is enabled, for the duration of zsh initialization standard input is redirected
+to `/dev/null` and a standard output with standard error are redirected to a temporary file. Once zsh
```
Comment (romkatv) [inline README.md]: This doesn't look right.

File: README.md [hunk 4]
```diff
@@ -308,7 +308,7 @@ If the `echo` command prints `❯` but the cursor is still in the wrong place, i
 
 If this doesn't help, add `unset ZLE_RPROMPT_INDENT` at the bottom of `~/.zshrc`.
 
-Still having issues? Run the following command to diagnose the problem:
+Still, having issues? Run the following command to diagnose the problem:
```
Comment (romkatv) [inline README.md]: This doesn't look right.

File: README.md [hunk 5]
```diff
@@ -350,7 +350,7 @@ culprits. Open an issue if you get stuck.
 ```
 
 This is usually caused by a terminal bug or misconfiguration that makes it print ambiguous-width
-characters as double-width instead of single width. For example,
+characters as double-width instead of a single width. For example,
```
Comment (romkatv) [inline README.md]: This doesn't look right.

File: README.md [hunk 6]
```diff
@@ -426,15 +426,15 @@ See also: [How do I change the format of Git status?](#how-do-i-change-the-forma
 
 ### How do I change the format of Git status?
 
-To change the format of Git status, open `~/.p10k.zsh`, search for `my_git_formatter` and edit its
+To change the format of Git status, open `~/.p10k.zsh`, Search for `my_git_formatter` and edit its
```
Comment (romkatv) [inline README.md]: This doesn't look right.

File: README.md [hunk 7]
```diff
@@ -426,15 +426,15 @@ See also: [How do I change the format of Git status?](#how-do-i-change-the-forma
...
 source code.
 
 ### How do I add username and/or hostname to prompt?
 
 When using *Lean*, *Classic* or *Rainbow* style, prompt shows `username@hostname` when you are
 logged in as root or via SSH. There is little value in showing `username` or `hostname` when you are
 logged in to your local machine as a normal user. So the absence of `username@hostname` in your
-prompt is an indication that you are working locally and that you aren't root. You can change it,
+prompt is an indication that you are working locally and that you aren't rooted. You can change it,
```
Comment (romkatv) [inline README.md]: This doesn't look right.

File: README.md [hunk 8]
```diff
@@ -496,62 +496,4 @@ Running this command with `~/powerlevel10k` as the current directory on the same
 [prompt benchmark](#is-it-really-fast) takes 29 seconds (29 ms per invocation). This is about 6
 times faster than powerlevel9k/master and 17 times faster than powerlevel9k/next.
 
-### Does Powerlevel10k always render exactly the same prompt as Powerlevel9k given the same config?
```
Comment (romkatv) [inline README.md]: Why this change?

**Comment by iamrajiv** [comment]
@romkatv I had used Grammarly to check typo mistakes so it suggested the changes what I made earlier

**Comment by romkatv** [comment]
Thanks! Merged.
